app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,49 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name, autofocus: true, autocomplete: "name" %>
+  </div>
+  <div class="field">
+    <%= f.label :photo %><br />
+    <%= f.text_field :photo, autofocus: true, autocomplete: "photo" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :bio %><br />
+    <%= f.text_field :bio, autofocus: true, autocomplete: "bio" %>
+  </div>
+  <div class="field">
+    <%= f.hidden_field :post_counter, value: 0%>
+  </div>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div><%= f.label :email_confirmation %><br />
+  <%= f.email_field :email_confirmation %></div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] Upon clicking on the `sign up` button, the app crashes with the following screen on the screenshot, would you address it?
![Screenshot from 2022-01-25 18-07-45](https://user-images.githubusercontent.com/56886064/151014932-2d75a36e-4244-4572-9cee-38efeb898950.png)

Reply:
Hi @descholar-ceo, we have leave a comment about this, we need some help to fix that problem, we have spend like 4 hours trying to solve, could you help us please ? 😄

Reply:
@ramiaquista Let me double check what could be the potential cause please

Reply:
Sure no problem, I have tried on my project to do it ( MacOS )and it work all perfectly, but in this project that it's on Windows Os and currently using a lower version of ruby it's crashing without sense. There is a possibility that we do it in another repository back, that we know it works 100% because we have spent a whole day with compatibility problems for the ruby version, I would appreciate it very much if it is possible. Thanks @descholar-ceo 😟

Reply:
@ramiaquista Do you mind sending me the link to that other repo which works well?

Reply:
@descholar-ceo Exactly we do pair programming on a repo that work's perfectly without versions incompatibilities issues. It's ok ?? And we should ask for another review

Reply:
Do you agree with that ? Because I now that it's not the best way but it's a real big problem about versions of ruby and gems incompatibilities, and OS that we can't handle and manage to solve 😢

Reply:
I will wait to your answer to start working on a new repo @descholar-ceo 😄

Reply:
Give me few minutes, I will give you a proper answer

Reply:
ok, thank's for all! 😄